src/edu/csus/ecs/pc2/ui/ShadowControlPane.java

```diff
@@ -987,6 +987,34 @@ public void run() {
         
     }
     
+    public static void addToConnectTableEntry(ShadowStatus stat, String msg)
```

Comment:
Since the code body of `addToConnectTableEntry() `is the same as `addConnectTableEntry()`, it probably makes sense to make `addConnectTable ``public static`, remove `addToConnectTableEntry`, and change the reference to `addToConnectTableEntry `in `RemoteEventFeedMonitor `to `addConnectTableEntry`.

Reply:
pushed fix, removed 'static' (and some other unneeded methods)